Module.yaml

```diff
@@ -0,0 +1,125 @@
+version: 1.0.0
+name: OPCUASubscribe
```

Comment:
add spacebar in the name `OPC UA Subscribe`

Reply:
updated.